lib/mayaUsd/utils/utilFileSystem.cpp

```diff
@@ -191,18 +197,35 @@ std::string UsdMayaUtilFileSystem::getPathRelativeToProject(const std::string& f
     //       preserve paths entered manually with relative folder ("..")
     //       by keping an absolute path with ".." embedded in them,
     //       so this works even in this situation.
-    const auto pos = fileName.find(projectPath.asChar());
+    const auto pos = fileName.rfind(projectPath, 0);
     if (pos != 0)
         return {};
 
-    auto relativePathAndSuccess = makePathRelativeTo(fileName, projectPath.asChar());
+    auto relativePathAndSuccess = makePathRelativeTo(fileName, projectPath);
```

Comment:
Can we use the `makeProjectRelatedPath` method? Seems like `makeProjectRelatedPath` handles all the checkings and returns the related path

Reply:
makeProjectRelatedPath cerates an *absolute* path, not a relative path. (I know the name of the function is awkward, because the function makes an absolute path, but relative to the absolute project path.)

Reply:
No because this function wants to make a true *relative* path, but makeProjectRelativePath makes an *absolute* path, but relative to the absolute path of the project.

Reply:
Oh I see